### PR TITLE
feat: added fastify-sensible to project template (#312)

### PR DESCRIPTION
**Pull request by alemagio**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: templates/app-ts/src/plugins/httpErrors.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import fp from 'fastify-plugin'
+import sensible, { SensibleOptions } from 'fastify-sensible'
+
```
Comment (mcollina) [inline templates/app-ts/src/plugins/httpErrors.ts:3]: this file should be called sensible instead

File: templates/app/plugins/httpErrors.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict'
+
```
Comment (mcollina) [inline templates/app/plugins/httpErrors.js:2]: same

**Review by mcollina — APPROVED** [review]
lgtm

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**